huaweicloud/services/ces/resource_huaweicloud_ces_alarm_template.go

```diff
@@ -0,0 +1,383 @@
+			StateContext: schema.ImportStatePassthroughContext,
+		},
+
+		Schema: map[string]*schema.Schema{
+			"region": {
+				Type:     schema.TypeString,
+				Optional: true,
+				Computed: true,
+				ForceNew: true,
+			},
+			"name": {
+				Type:        schema.TypeString,
+				Required:    true,
+				Description: `Specifies the name of the CES alarm template.`,
+			},
+			"policies": {
+				Type:        schema.TypeList,
+				Elem:        AlarmTemplatePolicySchema(),
+				Required:    true,
+				Description: `Specifies the policy list of the CES alarm template.`,
+			},
+			"description": {
+				Type:        schema.TypeString,
+				Optional:    true,
+				Computed:    true,
+				Description: `Specifies the description of the CES alarm template.`,
+			},
+			"delete_associate_alarm": {
+				Type:        schema.TypeBool,
+				Optional:    true,
+				Description: `Specifies whether delete the alarm rule which the alarm template associated with`,
+			},
+			"type": {
+				Type:        schema.TypeString,
+				Computed:    true,
+				Description: `Indicates the type of the CES alarm template.`,
+			},
+			"association_alarm_total": {
+				Type:        schema.TypeInt,
+				Computed:    true,
+				Description: `Indicates the total num of the alarm that associated with the alarm template.`,
+			},
+		},
+	}
+}
+
+func AlarmTemplatePolicySchema() *schema.Resource {
+	sc := schema.Resource{
+		Schema: map[string]*schema.Schema{
+			"namespace": {
+				Type:        schema.TypeString,
+				Required:    true,
+				Description: `Specifies the namespace of the service.`,
+			},
+			"dimension_name": {
+				Type:        schema.TypeString,
+				Required:    true,
+				Description: `Specifies the Resource dimension.`,
+			},
+			"metric_name": {
+				Type:        schema.TypeString,
+				Required:    true,
+				Description: `Specifies the alarm metric name.`,
+			},
+			"period": {
+				Type:        schema.TypeInt,
+				Required:    true,
+				Description: `Specifies the judgment period of alarm condition.`,
+			},
+			"filter": {
+				Type:        schema.TypeString,
+				Required:    true,
+				Description: `Specifies the data rollup methods.`,
+			},
+			"comparison_operator": {
+				Type:        schema.TypeString,
+				Required:    true,
+				Description: `Specifies the comparison conditions for alarm threshold.`,
+			},
+			"value": {
+				Type:        schema.TypeInt,
+				Required:    true,
+				Description: `Specifies the alarm threshold.`,
+			},
+			"unit": {
+				Type:        schema.TypeString,
+				Optional:    true,
+				Computed:    true,
+				Description: `Specifies the unit string of the alarm threshold.`,
+			},
+			"count": {
+				Type:        schema.TypeInt,
+				Required:    true,
+				Description: `Specifies the number of consecutive triggering of alarms.`,
+			},
+			"suppress_duration": {
+				Type:        schema.TypeInt,
+				Required:    true,
+				Description: `Specifies the alarm suppression cycle.`,
+			},
+			"level": {
```

Comment:
this parameter is called `alarm_level` in ces_alarmrule

Reply:
it has been change to alarm_level